system-services/usb-rauc-install.service

```diff
@@ -0,0 +1,9 @@
+[Unit]
+Description=Meticulous manual updater
+Requires=rauc.service
+
+[Service]
+Type=simple
+EnvironmentFile = /tmp/rauc_install/env_file

```

Comment:
Usually environment files are used as configuration and persistent. Putting them in /tmp is a bit non-canonical

Reply:
My idea with that is just to have a way to send the information from the udev script to the service script, and as I don't want it to be persistent placed it on /tmp

I look for another way to buffer that information and send it across, maybe dubs again :D